### PR TITLE
Add ability to edit organizations, closes #213

### LINKED ISSUE #213: Coyote Scavenger Hunt - updating organization name
So I realized that because we are MCA Chicago in Coyote our name is not displayed as Museum of Contemporary Art Chicago in the Coyote Scavenger Hunt interface. Can this be updated? I'm ok if solution is a hard code on the scavenger hunt side.

screenshot of organization, and side by side layout of scavenger hunt design:
screen shot 2018-09-13 at 12 03 51 pm
screen shot 2018-09-13 at 12 01 22 pm

### PR DESCRIPTION
**Pull request by stacietaylorcima**
**This PR allows the owner of an organization the ability to edit the organization by clicking the 'Edit' button on the organizations index view.** 

<hr>

I'm not super happy about the authorization line I added to the view, but followed Pundit's documentation to get it working. Ideally, I'd be able to call: 
```
- if policy(organization_user, org).edit? 
```
because you can call `policy` and pass through a user and a record then see if it is authorized to take an action. But, for some reason (that I could not figure out after an embarrassing amount of searching) the organization was not being passed through here and was resulting in Pundit returning a NilClass policy instead of OrganizationPolicy. So I had to use this line to instantiate a new instance and manually pass through the `current_user` and `org`: 
```
- if OrganizationPolicy.new(Coyote::OrganizationUser.new(current_user, org), org).edit?
```

Does anyone have thoughts on this? I'm happy to create an issue that will lead me to opening this back up and reinvestigating someday, but I think that it is working fine for now and I should move on to other issues since I'm pretty stumped. 

**Pundit Documentation:** https://github.com/varvet/pundit#policies

### CODE REVIEW

**Comment by stacietaylorcima** [comment]
@flipsasser Can I get your review on this as soon as you have time? Thanks so much!

File: app/views/organizations/index.html.slim [hunk 1]
```diff
@@ -1,8 +1,19 @@
-h1= title 'Your Organizations'
+table
+  caption
+    h1= title 'Your Organizations'
+  thread
```
Comment (flipsasser) [inline app/views/organizations/index.html.slim]: This is a typo. Should be `thead`

Comment (stacietaylorcima) [reply app/views/organizations/index.html.slim]: Oh my goodness. Fixed, committed, pushed! Thanks! @flipsasser

**Comment by stacietaylorcima** [comment]
@flipsasser Can you look into authorization issues we're having. Anna is the owner of the MCA organization, yet the edit button/ability to edit an organization is not available to her. You can see details above of what I did to implement the ability to edit organizations. I had to put a little workaround in to make the authorizations work. I think there are some authorization issues at large though that investigating this might uncover.

**Comment by alavatelliMCA** [comment]
Hey all, Sina and I spoke, if when Sina and Flip talk this isn't resolved. Below is our name and ideally how it would display in the hunt (note line breaks)

Museum of Contemporary Art Chicago

I am reviewing with designer re: line breaks, but I think we'll let it break where it needs to.

**Comment by sinabahram** [comment]
Anna, please confirm that you have the power ot change this now?

**Pull request merged**